python/pyspark/pandas/tests/data_type_ops/test_binary_ops.py

```diff
@@ -152,7 +152,7 @@ def test_from_to_pandas(self):
         data = [b"1", b"2", b"3"]
         pser = pd.Series(data)
         psser = ps.Series(data)
-        self.assert_eq(pser, psser.to_pandas())
+        self.assert_eq(pser, psser._to_pandas())
```

Comment:
Just for my info, what is the difference in the old and new method, anything functional?

Reply:
the only difference between `to_pandas()` and `_to_pandas()` is the [warning](https://github.com/apache/spark/blob/7e5a87f0128a7679490b89294ac75e8af0856d1f/python/pyspark/pandas/frame.py#L5406-L5409).